benchmarks/experiment_runner.py

```diff
@@ -235,20 +235,25 @@ def prepare_inputs(self, example_inputs, should_randomize_input):
       inputs_list.append(inputs)
     return inputs_list
 
-  def dump_profile_info(self, prof, model_name):
+  def dump_profile_info(self, prof, benchmark_model, benchmark_experiment):
     assert prof is not None, 'Expecting profiler to be defined!'
     if not self._args.profile_cuda_dump:
       logger.warning(
           'Profiling enabled, but dumping tracing/kernel summary disabled.')
       return
 
-    file_path = f"/tmp/{model_name}-profile"
+    with_configuration_suffix = lambda name, ext: f"{name}-{benchmark_model.filename_str}-{benchmark_experiment.filename_str}.{ext}"
+    model_name = benchmark_model.model_name
+    file_path = os.path.join(self._args.profile_cuda_dump, model_name)
     os.makedirs(file_path, exist_ok=True)
-    prof.export_chrome_trace(os.path.join(file_path, "trace.json"))
+    prof.export_chrome_trace(
+        os.path.join(file_path, with_configuration_suffix("trace", "json")))
 
     kernel_dump = prof.key_averages().table(
         sort_by="cuda_time_total", row_limit=500)
-    with open(os.path.join(file_path, "kernel_dump.txt"), "a") as f:
+    with open(
+        os.path.join(file_path, with_configuration_suffix("kernel_dump",
```

Comment:
This is the PyTorch profile, right? Can we include `pt-profile` or something in the name to disambiguate form the ptxla metrics?

Reply:
Done.